README.md

```diff
@@ -370,35 +370,19 @@ restApiConfig:
 
 ## Step 9: Customize Model Deployment
 
-In the `ecsModels` section of `config.yaml`, configure the models you want to deploy.
+In the `ecsModels` section of `config.yaml`, allow our deployment process to pull the model weights for you.
 
-The configuration file will determine which models are deployed. In order to deploy an additional model or a
-different model the only required change is to the configuration file, as long as it is compatible with the inference
-container. Specifically, see the `ecsModels` section of the [config.yaml](./config.yaml) file.
-Here we define the model name, if we want to deploy, the type of instance we want to deploy to, the type of model
-(textgen or embedding), the inference container and then the containerConfig. There are many more parameters for the
-ecs models, many for autoscaling and health checks. However, let's focus on the model specific ones:
+During the deployment proces, lisa will optionally attempt to download your model weights if you specify an optional`ecsModels`
+array, this will only work in non ADC regions. Specifically, see the `ecsModels` section of the [config.yaml](./config.yaml) file.
```

Comment:
technically this file doesn't exist in the repo. let's just use backticks to display as `config.yaml` or something to indicate it's important, but let's not link directly to it, otherwise this is a 404 (which i realize is already in the readme... this is already a bug in the readme 😬 )